### PR TITLE
Try to not fail when you don't have write permissions

### PR DESCRIPTION
**Pull request by IAlibay**
Changes:
- micromamba for speed
- check user permissions actions
- artifact uploading

### CODE REVIEW

**Comment by IAlibay** [comment]
> https://github.com/MDAnalysis/mdanalysis-sphinx-theme/actions/runs/6067130313/job/16458435672?pr=38#step:2:7

Yikes, maybe we don't use _that_ action.. or we maintain it.

**Comment by IAlibay** [comment]
🙈 yikes we have 2 GB of caches: https://github.com/MDAnalysis/mdanalysis-sphinx-theme/actions/caches

File: .github/workflows/gh_pages.yml [hunk 1]
```diff
@@ -56,6 +60,13 @@ jobs:
           cd docs
           make html
 
+      - name: upload doc artifact
```
Comment (IAlibay) [inline .github/workflows/gh_pages.yml]: See artifacts here: https://github.com/MDAnalysis/mdanalysis-sphinx-theme/actions/runs/6067130313

**Comment by lilyminium** [comment]
To be honest, what would you think of just removing this action? I think I'm happy with ReadTheDocs for now only.

**Comment by lilyminium** [comment]
That would avoid both storing PRs on our domain and the artifact cache issue.

**Comment by IAlibay** [comment]
> To be honest, what would you think of just removing this action? I think I'm happy with ReadTheDocs for now only.

I mean you're the person in the know here, if you're happy with removing it then I would just happily agree with you.

Although don't you still need the action for docs publishing on release? So just remove the PR triggers / bits?

**Comment by lilyminium** [comment]
Yes please!

**Review by lilyminium — APPROVED** [review]
LGTM!

**Comment by IAlibay** [comment]
I guess we'll know if it fails on merge 🙃

**Comment by IAlibay** [comment]
All good, CI didn't fail.

**Pull request merged**